src/bacnet/basic/object/csv.c

```diff
@@ -123,6 +123,44 @@ uint32_t CharacterString_Value_Create(uint32_t object_instance)
     return object_instance;
 }
 
+/**
+ * @brief Delete an object and its data from the object list
+ * @param  object_instance - object-instance number of the object
+ * @return true if the object is deleted
+ */
+bool Characterstring_Value_Delete(uint32_t object_instance)
+{
+    bool status = false;
+    struct object_data *pObject = NULL;
+
+    pObject = Keylist_Data_Delete(Object_List, object_instance);
+    if (pObject) {
+        free(pObject);
+        status = true;
+    }
+
+    return status;
+}
+
+/**
+ * @brief Cleans up the object list and its data
+ */
+void Characterstring_Value_Cleanup(void)
+{
+    struct object_data *pObject;
```

Comment:
might as well initilialize this to NULL, not because it is an issue but just for consistency with line 134

Reply:
Resolved.